packages/accordions/src/styled/accordion/StyledPanel.ts

```diff
@@ -17,38 +22,56 @@ interface IStyledPanel {
 
 const COMPONENT_ID = 'accordions.panel';
 
-const paddingStyles = (props: IStyledPanel & ThemeProps<DefaultTheme>) => {
-  const { base } = props.theme.space;
+const colorStyles = (props: IStyledPanel & ThemeProps<DefaultTheme>) => {
+  const { theme, isBare } = props;
+
+  return css`
+    border-bottom-color: ${isBare ? 'transparent' : getColor('neutralHue', 300, theme)};
+  `;
+};
+
+const sizeStyles = (props: IStyledPanel & ThemeProps<DefaultTheme>) => {
+  const { theme, isCompact, isExpanded } = props;
+  const { base } = theme.space;
   let paddingTop = base * 2;
   let paddingHorizontal = base * 5;
   let paddingBottom = base * 8;
 
-  if (props.isCompact) {
+  if (isCompact) {
     paddingTop = base * 2;
     paddingHorizontal = base * 3;
     paddingBottom = base * 4;
   }
 
-  if (props.isExpanded === false) {
+  if (isExpanded === false) {
     paddingTop = 0;
     paddingBottom = 0;
   }
 
   return css`
-    transition: ${props.isAnimated && 'padding 0.25s ease-in-out'};
+    grid-template-rows: ${isExpanded ? 1 : 0}fr;
+    border-bottom: ${theme.borders.sm};
```

Comment:
Recommend using `border-bottom-width` here.

Reply:
Hmm setting `border-bottom-width` didn't give us the same styling as `isBare` styling which needs additional properties. Setting `border-bottom` seems to be the most economical here.
![Screenshot 2023-08-07 at 3 09 31 PM](https://github.com/zendeskgarden/react-components/assets/12474067/e205061b-99f7-482f-98b8-89a8c867700b)

Reply:
We tend to get desired results by separating out all of the `border` shorthand – `border-color`, `border-width`, and `border-style`. See https://github.com/zendeskgarden/react-components/blob/main/packages/drag-drop/src/styled/dropzone/StyledDropzone.ts for reference.